app/components/muuri-grid-component.js

```diff
@@ -0,0 +1,16 @@
+import Component from '@ember/component';
+import Muuri from 'muuri';
+
+export default Component.extend({
+  classNames: ['muuri-grid-component'],
+  didRender() {
```

Comment:
This this work on `didInsertElement`?

`didRender` is called multiple times:
https://guides.emberjs.com/release/components/the-component-lifecycle/#toc_making-updates-to-the-rendered-dom-with-didrender

Reply:
We should probably tear this binding down too on willRemoveElement hook